docs/develop/05_dApp/01_welcome.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This empty screen seems unnecesary

Reply:
I think we can utilise it for the overview.